projects/igniteui-angular/src/lib/drop-down/drop-down.component.ts

```diff
@@ -547,6 +547,38 @@ export class IgxDropDownComponent extends IgxDropDownBaseDirective implements ID
         }
     }
 
+     /**
+     *
+     * @hidden
+     * @internal
+     * @param newSelection
+     */
+     public selectItemWithoutEvent(item: IgxDropDownItemBaseDirective) {
```

Comment:
This seems to be a complete repeat of the `selectItem` method just without the emit, which seems redundant to me. If `selectItem` is already internal, how about we simply add an optional `emit = true` param to it and call it with false for the EFS case?
Also kind of curios what trouble the event causes, though I can sort of imagine :)